packages/p/poco/xmake.lua

```diff
@@ -21,7 +21,7 @@ package("poco")
     end
     add_configs("mysql", {description = "Enable mysql support.", default = false, type = "boolean"})
     add_configs("postgresql", {description = "Enable postgresql support.", default = false, type = "boolean"})
-    add_configs("odbc", {description = "Enable odbc support.", default = is_plat("windows") or not is_plat("mingw"), type = "boolean"})
+    add_configs("odbc", {description = "Enable odbc support.", default = is_plat("windows") and not is_plat("mingw", "msys"), type = "boolean"})
```

Comment:
? windows -> true, other -> always false

Reply:
the original setting for other platform is false. And MingW and msys platform need this be false to pass the compile.

Reply:
the original settings:  windows-> true, other -> false
the current settings:  windows -> true, other -> false

I don't see any difference.